app/build.gradle

```diff
@@ -117,6 +113,32 @@ android {
     }
 }
 
+static def addLocalProperties(buildType, localPropertiesFile) {
```

Comment:
How is thi function helping out? The local properties should be consistent and not change for different envs. Local properties are only for local dev work and shouldn't be used this way. We already replace these values while making a prod/beta build from the gh workflows I wrote. This use case is not useful for us.

Reply:
It's the same thing as debug build, since beta build will be used for local development only, this function is just a code reduction measure, nothing else. The functionality remains similar.

Reply:
What makes you think that we want to use beta builds only for local testing? We should release stuff on the go and when on beta builds, use beta apis and not hit prod server.

Reply:
Some feature are limited in beta. Such as socket connections.

Reply:
If there are limitations in beta then we need to ask the backend team to resolve those. We can't have a build with some apis hiting the beta apis and some hitting the prod apis. We need to maintain consistency